kommunicate/src/main/java/com/applozic/mobicomkit/api/account/user/MobiComUserPreference.java

```diff
@@ -968,4 +969,17 @@ public void setLoggedUserDeletedFromDashboard(boolean deletedFromDashboard) {
             sharedPreferences.edit().putBoolean(LOGGED_USER_DELETE_FROM_DASHBOARD, deletedFromDashboard).commit();
         }
     }
+
+    public boolean isAgentRouting() {
+        if(sharedPreferences != null) {
+            return sharedPreferences.getBoolean(AGENT_ROUTING, true);
+        }
+        return false;
+    }
+
+    public void setAgentRouting(boolean isAgentRouting) {
```

Comment:
Should we change this to `notifyEverybody`? If that's what we are doing then it will be better to set the same here and in other methods.